### PR TITLE
Add Docker support

### PR DESCRIPTION
**Pull request by djjudas21**
This PR adds a `Dockerfile` and updates documentation so a Docker image can be built.

I also allowed some of the args to be set by env var, which will make it much easier to use this tool in Kubernetes.

### CODE REVIEW

File: photo_prism_yaml_to_exif.pl [hunk 1]
```diff
@@ -33,14 +33,14 @@
       }
     , yaml_dir              => {
           type      => '=s'
-        , env       => '-'
+        , env       => 'YAML_DIR'
         , default   => cwd()
```
Comment (JiCiT) [inline photo_prism_yaml_to_exif.pl]: the env property affects _only_ the help message text, so for env vars to be useful, this line needs changed to:
```perl
, default => $ENV{'YAML_DIR'} || cwd()
```

File: photo_prism_yaml_to_exif.pl [hunk 2]
```diff
@@ -33,14 +33,14 @@
...
         , verbose   => 'Root directory with PhotoPrism YAML sidecar files'
         , order     => 2
       }
     , image_dir         => {
           type      => '=s'
-        , env       => '-'
+        , env       => 'IMAGE_DIR'
         , default   => cwd()
```
Comment (JiCiT) [inline photo_prism_yaml_to_exif.pl]: Like line 37, this needs changed to:
```perl
default = $ENV{'IMAGE_DIR'} || cwd()
```

**Review by JiCiT — CHANGES_REQUESTED** [review]
Thanks for the PR!  Just two small changes need made.  Please see comments.

File: photo_prism_yaml_to_exif.pl (hunk 1 above)
Comment (djjudas21) [reply photo_prism_yaml_to_exif.pl]: Ah yes. I think I was crediting the Getopt module with too much intelligence to go and get the env vars itself. Fixed now

File: photo_prism_yaml_to_exif.pl (hunk 2 above)
Comment (djjudas21) [reply photo_prism_yaml_to_exif.pl]: Fixed

**Pull request merged**